auto_inertial_params/proposal.md

```diff
@@ -0,0 +1,127 @@
+* **Status**: Draft
+* **SDFormat Version**: 1.11
+* **`libSDFormat` Version**: 14.X 
+
+## Introduction
+
+This proposal suggests adding new attributes and elements to support the automatic calculation of Moments of Inertia and Products of Inertia for a link in SDFormat 1.11. It also proposes adding support for parsing these elements and attributes in libsdformat14.
+
+Setting physically plausible values for inertial parameters is crucial for an accurate simulation. However, these parameters are often complex to comprehend and visualize. Therefore, incorporating native support for the automatic calculation of inertial parameters of a link through the SDF specification would enable simulators that utilize SDFormat to provide an enhanced and smoother workflow.
+
+## Document summary
+
+The proposal includes the following sections:
+
+* [Motivation](#motivation): A short explanation to provide context reagrding the problem statement and the need for this feature
+* [User Perspective](#user-perspective): Describes the current usage and the proposed usage by describing the terms to be added in the SDFormat specification
+* [Proposed Implmentation](#proposed-implementation): Detailed explanation of the proposed implmentation and modifications to be done in the C++ API of requried libraries like libsdformat. 
+
+## Motivation
+
+Physically plausible values for inertial parameters like mass, center of mass, moments of inertia, etc. are required for an accurate simulation. Such parameters are often difficult to visualize and a user may tend to enter wrong values for parameters This often leads to an incorrect simulation which is hard to debug.
+
+Currently, there are 2 major workflows used by the users to obtain the correct inertial parameters of their models:
+
+ * Using CAD softwares like [Fusion360](https://www.autodesk.in/products/fusion-360/overview?term=1-YEAR&tab=subscription) or [Solidworks](https://www.solidworks.com/). Many users design their robot models using such CAD software which usually provide plugins that automatically generates the URDF/SDF for their model. These plugins handle the calculation of the inertial parameters. For eg, Fusion360 provides the [Fusion2URDF](https://github.com/syuntoku14/fusion2urdf) plugin which automatically generates a URDF with all the inertial parameters.
+
+ * Another way is to use 3rd-party Mesh Processing Software like [Meshlab](https://www.meshlab.net/). Such softwares take the mesh file as an input and provide the inertial parameters as an output which can then be copied and pasted into the URDF/SDF file. This is also the method that was suggested in official [Classic Gazebo docs](https://classic.gazebosim.org/tutorials?cat=build_robot&tut=inertia).
+
+Both of these ways create a dependency on external software and might be complicated for beginners. Integrating this directly in Gz Sim would provide a smoother and user-friendly workflow.
```

Comment:
- Not sure if we should be talking about Gazebo here?
- Rephrase "smoother and user-friendly workflow" to the specific impact on maintaining physically plausible and accurate simulation